bin/utility/utility.py

```diff
@@ -44,6 +44,60 @@ class IETF(Enum):
     EXAMPLE = auto()
 
 
+def list_of_files_of_particular_extensions_in_dir(
```

Comment:
I would use shorter name for this method, like: `list_files_by_extension`

Reply:
renamed to ```list_files_by_extensions```, so it's obvious from the name that not only one extension can be specified

Reply:
perfect, that's true